IBF-Typhoon-model/src/typhoonmodel/pipeline.py

```diff
@@ -73,22 +77,40 @@ def main(path,remote_directory,typhoonname,LonMin,LatMin,LonMax,LatMax):
     Activetyphoon=Check_for_active_typhoon.check_active_typhoon()
     TEST_REMOTE_DIR = '20210421120000'
     remote_dir = remote_directory
-    if not Activetyphoon:
-      logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
-      if remote_dir is None:
-         remote_dir = TEST_REMOTE_DIR
-         #remote_dir='20210421120000' #for downloading test data otherwise set it to None
-      Activetyphoon=[typhoonname]  #name of typhoon for test
-      #logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
-    elif remote_directory is not None:
-      logging.info(f"There is an active typhoon, but the user has requested to run for typhoon{typhoonname}")
-      #remote_dir='20210421120000' #for downloading test data otherwise set it to None
-      Activetyphoon=[typhoonname]  #name of typhoon for test
-      logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")    
+    if not Activetyphoon and debug_flag:
```

Comment:
I would change this to only `if debug_flag`. There might be times when there is an active typhoon, but you still want to test the pipeline with the debugging option.